ports/libxmlmm/CMakeLists.txt

```
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.8)
+project(libxmlmm CXX)
+
+set(CMAKE_SHARED_LIBRARY_PREFIX)
+set(CMAKE_STATIC_LIBRARY_PREFIX)
+
+find_package(libxml2 REQUIRED)
```

```suggestion
find_package(LibXml2 REQUIRED)
```
It seems that we should use this name.

The usage for `libxml2` is like this:
```
find_package(LibXml2 REQUIRED)
target_include_directories(main PRIVATE ${LIBXML2_INCLUDE_DIR})
target_link_libraries(main PRIVATE ${LIBXML2_LIBRARIES})
```

---

Yes, I was wondering why the linux build failed. Sorry I figured it out before looking at this PR.